src/dgipy/dgidb.py

```diff
@@ -220,6 +220,75 @@ def get_drug_applications(
     return result
 
 
+def get_clinical_trials(
+    terms: str | list,
+) -> pd.DataFrame:  # TODO: Better error handling for new_row?, use_pandas=False
+    """Perform a look up for clinical trials data for drug or drugs of interest
+
+    :param terms: drug or drugs of interest
```

Comment:
are there any guarantees about what form a given term takes? are we hoping that the label provided by the normalizer is the term used by the trial? I just tried imatinib vs gleevec and I think they are both returning the same results

Reply:
In its current state, yes just essentially hoping that the label the normalizer gives is the correct search term to use. I imagine whether or not it works on clinical trials end (i.e. imatinib vs gleevec) will be determined by how well the clinical trial entries themselves have been tagged/organized for the appropriate interventions? The imatinib and gleevec example I can see someone remembering to associate both of those names with any clinical trial just because its so well studied, but maybe this isn't always the case and maybe some aliases will differ in results.